tests/conversion_script/test_main.py

```diff
@@ -80,8 +80,8 @@ def test_main_success_c2r_installed(
     assert mock_update_insights_inventory.call_count == 1
     assert mock_gather_json_report.call_count == 1
     assert mock_find_highest_report_level.call_count == 1
-    assert mock_gather_textual_report.call_count == 1
-    assert mock_transform_raw_data.call_count == 1
+    assert mock_gather_textual_report.call_count == 0
+    assert mock_transform_raw_data.call_count == 0
```

Comment:
The tests had to be changed because the way we are mocking the run_convert2rhel function is that we are returning a return code 0, meaning that we won't collect the textual report and not transform the raw data to display in insights.